cloudzero/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: CloudZero
+
+## Overview
+
+This check monitors [CloudZero][1].
+
+## Setup
+
+### Installation
+
+To install the CloudZero check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/python/)
+ on any machine.
+
+2. Run `ddev release build cloudzero` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/cloudzero/dist/<ARTIFACT_NAME>.whl`.
```

Comment:
Please update this section to represent how to install your integration - the user will navigate to the integration tile, click install, click connect accounts, and then go through the OAuth flow (please describe how the OAuth flow works on your end once the user is brought to the onboarding_url).